### ISSUE TITLE
Link to "This version" in editor's draft leads to 404

### ISSUE THREAD

**Issue by mfru** [w3c-social/activitypump#442]
Currently, the link to "This version" on https://w3c.github.io/activitypub/ is leading to a [404](https://www.w3.org/community/reports/socialcg/CG-FINAL-activitypub-20240104/).

Unfortunately, I wasn't able to find the exact link to the version from January 4th.

**Comment by evanp** [comment]
Yes, it looks like we need to have a permanent URL for this. I will take it as a task to determine how we get that.
